### PR TITLE
Feature/250: Bug Fixes and Improvements

### PR DESCRIPTION
**Pull request by sherrif10**
PR  https://sd-vxnaid.atlassian.net/browse/VRVU-250

### CODE REVIEW

File: app/src/main/res/values/strings.xml
Comment (druchniewicz) [inline app/src/main/res/values/strings.xml:1]: what are these changes?

File: app/src/main/java/com/jnj/vaccinetracker/common/data/managers/ParticipantManager.kt [hunk 1]
```diff
@@ -52,7 +52,6 @@ class ParticipantManager @Inject constructor(
     /**
      * Gets the person image for a person and decodes the base64 response to a byte array
      */
-    @Suppress("BlockingMethodInNonBlockingContext")
```
Comment (druchniewicz) [inline app/src/main/java/com/jnj/vaccinetracker/common/data/managers/ParticipantManager.kt]: why was this line removed?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -89,12 +90,12 @@
     <string name="participant_flow_participant_id_hint">Scan Child ID QR Code</string>
     <string name="participant_flow_participant_id_scanned">The format of the ID is not supported</string>
     <string name="participant_flow_participant_id_description">Please scan the Child vaccination card with a QR-Code or manually enter the Child ID number:</string>
-    <string name="participant_flow_participant_id_label_manual_entry">Manual ID Entry</string>
-    <string name="participant_flow_participant_id_input_hint">#1234567890</string>
+    <string name="participant_flow_participant_id_label_manual_entry">Manual ID Entry or Search by Name of the Mother </string>
```
Comment (sherrif10) [inline app/src/main/res/values/strings.xml:93]: Hello @druchniewicz  , we need to add a query that searches name of the mother here. You can guide me on this. this is something that came up today.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -455,18 +456,19 @@
...
     <string name="visits_overview_end_date_label">End Date</string>
     <string name="visits_overview_not_set_label">Not set</string>
-    <string name="visits_overview_search_hint">Search by Client ID or name</string>
+    <string name="visits_overview_search_hint">Search by Child ID or name</string>
     <string name="visits_overview_visit_date_column_label">Visit Date</string>
-    <string name="visits_overview_client_id_column_label">Client ID</string>
-    <string name="visits_overview_client_name_column_label">Client Name</string>
+    <string name="visits_overview_client_id_column_label">Child ID</string>
+    <string name="visits_overview_client_name_column_label">Child Name</string>
+    <string name="visits_overview_client_mothers_name_column_label">Mothers Name</string>
```
Comment (sherrif10) [inline app/src/main/res/values/strings.xml:463]: We need to also add query for mothers here.
